ledger/sandbox/src/main/resources/db/migration/V1__Init.sql

```diff
@@ -19,22 +19,22 @@ CREATE TABLE ledger_entries
   -- table represents a single unified stream of events and partitioning
   -- it across tables would be more inconvient. We might revise this in
   -- the future.
-  typ                   varchar                      not null,
+  typ                   varchar(1024)                      not null,
```

Comment:
@rickibm I presume this is because DB2 does not have a variable-length `varchar`. Does it have _any_ variable-length text type?